splinterd/src/config/command_line.rs

```diff
@@ -107,9 +107,20 @@ impl PartialConfigBuilder for CommandLineConfig {
 mod tests {
     use super::*;
 
+    use std::env;
+    use std::fs;
+    use std::time::Duration;
+
     use clap::ArgMatches;
 
-    /// Example configuration values.
+    use crate::config::{Config, ConfigBuilder};
+
+    use crate::config::{DefaultConfig, EnvVarConfig, TomlConfig};
+
+    /// Path to existing example config toml files from the top-level Splinterd directory.
+    static TEST_TOML: &str = "sample_configs/splinterd.toml.example";
```

Comment:
We should de-couple "sample" files from the tests, because updating a sample config shouldn't break the tests.

Reply:
I added a commit to de-couple the sample files.